pandas/core/indexes/category.py

```diff
@@ -324,20 +324,19 @@ def _reverse_indexer(self):
     @Appender(_index_shared_docs['__contains__'] % _index_doc_kwargs)
     def __contains__(self, key):
         hash(key)
```

Comment:
This might be a really silly question, but what does this line do?

Reply:
It just ensures that mutables are not passes into the function....

I think it should probably be removed, but that line is in various other places as well, so maybe a seperate PR, that goes through all similar cases? Or I can just remove it.

Reply:
Well, you removed it in another part of the diff, hence why I'm asking.  Let's investigate for another time then, in which case I would put back the other one that you deleted.